File: benchmark/array2.py
#!/usr/local/bin/python3

base = 800

X = {}
for i in range(50):
  X[i] = str(i) + "." + str(i)

for j in range(base): 
  for i in range(49):
    X[i] = X[i+1]
  X[i] = X[0]

File: benchmark/array2.pl
#!/usr/local/bin/perl

$base = 800;

for ($i=0; $i<50; $i++) {
    $X{$i} = $i.".".$i;
}

for ($j=0; $j<$base; $j++) {
  for ($i=0; $i<49; $i++) {
    $X{$i} = $X{$i+1}
  }
  $X{$i} = $X{0}
}
